lldb/source/API/SBProcess.cpp

```diff
@@ -1255,6 +1255,95 @@ lldb::SBFileSpec SBProcess::GetCoreFile() {
   return SBFileSpec(core_file);
 }
 
+addr_t SBProcess::GetAddressMask(AddressMaskType type,
+                                 AddressMaskRange addr_range) {
+  LLDB_INSTRUMENT_VA(this, type, addr_range);
+  addr_t default_mask = 0;
```

Comment:
Would it be worth making this a constant (e.g. `LLDB_DEFAULT_ADDRESS_MASK`) so that callers can verify whether they were given a valid address mask?

Reply:
Wouldn't we need a `LLDB_INVALID_ADDRESS_MASK` as well? It can happen that the address mask is valid, and has a value of 0.

Reply:
A mask value of 0 would be an invalid address mask -- no bits are used for addressing.  The variable name "default_mask" was a poor choice.  But explicitly returning LLDB_INVALID_ADDRESS_MASK if we don't find any value might be the correct choice, it would mean you passed an invalid enum argument (or the SBProcess can't access its Process)